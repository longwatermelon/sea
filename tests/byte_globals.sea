let g1:byte = 'X';
let g2:byte = 5b;
let g3:byte;

fn main() -> int {
    g3 = g1 + g2;
    return g3;
}
