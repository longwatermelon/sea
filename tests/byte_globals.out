RETURN_CODE:93
STDOUT:
